static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -104,18 +104,20 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
     } = this.props;
     const {requestList, inviteRequests} = this.state;
 
+    const action = (
+      <Button
+        priority="primary"
+        size="small"
+        onClick={() => openInviteMembersModal({source: 'members_settings'})}
+        icon={<IconMail />}
+      >
+        {t('Invite New Members')}
```

Comment:
Thoughts on just "Invite member" or "Invite members" as the cta?

Reply:
err I mean kill the new